e3nn_jax/_src/s2grid_test.py

```diff
@@ -1,19 +1,27 @@
 import jax
 import numpy as np
 from e3nn_jax._src.s2grid import from_s2grid, irfft, rfft, to_s2grid
+from e3nn_jax._src.irreps_array import Irreps, IrrepsArray
 
 key = jax.random.PRNGKey(0)
 
 
-def test_s2grid_transforms():
-    res_alpha = 11
-    res_beta = 8
-    l = 3
+def test_s2grid_transforms(quadrature):
+    assert quadrature in ["soft", "gausslegendre"], "quadrature must be 'soft' or 'gausslegendre"
```

Comment:
Please decorate this function with pytest decorator

see an example here https://docs.pytest.org/en/6.2.x/parametrize.html